src/collective/lazysizes/profiles/uninstall/controlpanel.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<object name="portal_controlpanel">
+  <configlet
+      remove="True"
+      title="lazysizes"
+      action_id="lazysizes"
+      appId="collective.lazysizes"
```

Comment:
I think you only need the `action_id` here.

Reply:
I thought so too, but under Plone 4 you need all of these. (100% sure for the title, not 100% for the appId).
Plone 5 works only with action_id